theme/templates/resource-landing-page/modals.html

```diff
@@ -1775,6 +1775,33 @@ <h4 class="modal-title">Get file URL</h4>
             </div>
 
             <div class="modal-body">
+                <div id="warnTxtFileURL" class="row" style="display:none">
```

Comment:
I would move this inside the div below and give it the class `'col-sm-12'` so we get consisting padding. Typically, for layouts like this, you only need one `row` with its `col` children, unless it makes sense structurally to separate them.

Reply:
makes sense, thanks. Just pushed a commit to make this change